src/main/java/pages/DashboardPage.java

```diff
@@ -35,18 +34,21 @@ public void addNewDashboard(String dashboardName) {
         addNewDashboardButton.click();
         setDashboardName(dashboardName);
         clickDashboardModalButton(ADD_BUTTON);
-        waitUntil(() -> isAddedDashBoardDisplayed(dashboardName), 5, 1);
+        fluentWaitUntilElementPresent(getDashBoardTitleElement(dashboardName), 5);
         dataStorage.setDashboardName(dashboardName);
     }
 
     public boolean isDashBoardPageDisplayed() {
-        waitUntil(() -> isElementDisplayed(addNewDashboardButton), 5, 1);
+        fluentWaitUntilElementPresent(addNewDashboardButton, 5);
```

Comment:
you'll get timeout exception instead of false if element is not displayed

Reply:
not exactly, 
this step "        fluentWaitUntilElementPresent(addNewDashboardButton, 5);" is to wait 5 seconds
and then this step "isElementDisplayed(addNewDashboardButton);" has "webElement.isDisplayed()" inside, so by the end correct boolean is returned

Reply:
fixed